### ISSUE TITLE
I should click twice to enter edit mode

### ISSUE THREAD

**Issue by burakkilic** [GabiGrin/angular-editable-text#7]
Hi;

I should click twice to enter edit mode. How can I start to write text after first click?

**Comment by GabiGrin** [comment]
Hey,
I cannot recreate this. After the click it is also focused.
Which browser are you using? Does it happen in the demo (http://gabigrin.github.io/angular-editable-text/) too?

**Comment by burakkilic** [comment]
You are right. It works in your example demo. Something happens with my code but I could't fix. Thank you

UPDATE: It also doesn't work in this plunkr:

http://plnkr.co/edit/K7x7leZDAXvc3V0nM6nJ?p=preview

I tried in chrome.

**Comment by rhutchison** [comment]
I have the same issue in your plunkr and on my site.  Works fine in the demo.  I am using chrome for all 3 tests.

In the demo, he is using 1.2.x - in plunkr 1.4.x.  When I updated the plunkr to use 1.2.x, plunkr was working like the demo site.  Looks like something to do with angular upgrade.

I wrapped the .focus() line with a timeout and seems to have resolved it.

$timeout(function() { inputElm.focus(); });

**Comment by reppners** [comment]
Observed the same when using angular 1.4.x - any other way to solve this than the $timeout?

**Comment by pguetan** [comment]
Got the same issue. The $timeout workaround didn't fix it for me either.
